### PR TITLE
html pages created based on wireframes

### PR DESCRIPTION
**Pull request by cd0868**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<body>
+<!--Begin header-->
+<header>
+    <h2>Sushila Srinivasan</h2>
+    <h4>Freelance Web Developer</h4>
+    <nav><a href="index.html">Home</a> | <a href="portfolio.html">Portfolio</a> | <a href="ux.html">UX</a></nav>
+</header>
+<!--End header-->
+
+<!--Begin Content-->
```
Comment (axamei) [inline about.html]: I recommend the use of `<main>` to distinguish the main portion of your page.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  <li>Manipulate and update existing site content</li>
+  <li>Update store front catalogue content and copy</li>
+</ul>
+<h3>TRAINING & ADMINISTRATION</h3>
+<ul>
+  <li>Course list, Brochures, Training Material Development</li>
+  <li>Organizing Meetings, Venue, Training Classes</li>
+  <li>Tutoring and Career Counseling</li>
+</ul>
+<p>
```
Comment (axamei) [inline about.html]: If you do not intend to put text here, I would remove the paragraph element.

File: index.html [hunk 3]
```diff
@@ -7,7 +7,30 @@
   <title>Srinivasan Hello World 7420</title>
 </head>
 <body>
-  <h1>Hello World!</h1>
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/kZysLxfc_pc" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
-</body>
+  <!--Begin header-->
+  <header>
+      <h2>Sushila Srinivasan</h2>
+      <h4>Freelance Web Developer</h4>
```
Comment (axamei) [inline index.html]: You should not skip heading levels. Some of your pages do not have an `<h3>` tag.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Srinivasan Hello World 7420</title>
+</head>
+<body>
+<!--Begin header-->
+<header>
+    <h2>Sushila Srinivasan</h2>
+    <h4>Freelance Web Developer</h4>
+    <nav><a href="index.html">Home</a> | <a href="portfolio.html">Portfolio</a> | <a href="ux.html">UX</a></nav>
```
Comment (axamei) [inline about.html]: The navigation menu should be an unordered list.

**Review by axamei — CHANGES_REQUESTED** [review]

**Comment by cd0868** [comment]
I have made the required changes per your request.

**Review by axamei — APPROVED** [review]

**Pull request merged**